.travis.yml

```
@@ -0,0 +1,11 @@
+language: node_js
+sudo: false
+
+node_js:
+  - "10"
+  - "12"
+  - "14"
```

nit: sort the newest at the top
```suggestion
  - "14"
  - "12"
  - "10"
```

---

Updated in https://github.com/pkgjs/wiby/pull/12/commits/5b5e684f1704ea1bf553f717f69211a902e1f21e, but for the record I find this change entirely pointless.

Nits like this are distracting, discouraging and demotivating.

---

I'm sorry I caused you to feel this way; I'd love to understand more about why it's discouraging or demotivating. To me, this is just part of any code review. It being a "nit" just means it's not a blocker, and i'd be happy to explain why it's not pointless if you're interested.

---

The motivation for this probably matters more on https://github.com/nodejs/ci-config-travis/pull/6 than here.

---

In an imported config, the order is backwards; so i believe it's correct there :-)